op-acceptor/metrics/metrics.go

```diff
@@ -42,51 +44,119 @@ var (
 		"result",
 	})
 
-	acceptanceResults = promauto.NewGaugeVec(prometheus.GaugeOpts{
+	// Run-level metrics with run_id
+	testRunStatus = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: MetricsNamespace,
-		Name:      "acceptance_results",
-		Help:      "Result of acceptance tests",
+		Name:      "test_run_status",
+		Help:      "Status of test runs (value is always 1, results indicated by the 'result' label)",
 	}, []string{
 		"network_name",
 		"run_id",
 		"result",
 	})
 
-	acceptanceTestTotal = promauto.NewCounterVec(prometheus.CounterOpts{
+	testRunTestsTotal = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: MetricsNamespace,
-		Name:      "acceptance_test_total",
-		Help:      "Total number of acceptance tests",
+		Name:      "test_run_tests_total",
+		Help:      "Total number of tests in a run",
 	}, []string{
 		"network_name",
 		"run_id",
 	})
 
-	acceptanceTestPassed = promauto.NewCounterVec(prometheus.CounterOpts{
+	testRunTestsPassed = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: MetricsNamespace,
-		Name:      "acceptance_test_passed",
-		Help:      "Number of passed acceptance tests",
+		Name:      "test_run_tests_passed",
```

Comment:
Are you open to using a different name here (and on other metrics)? The term `test_run_tests` threw me off

Reply:
Certainly! What though? :)

I've updated the description of this PR with a summary of the metrics which provides context as to the naming.
But certainly open to renaming them; please let me know if you have a better idea.

Given your approval, I'll merge this for now so I can start dashboarding and experimenting with them, but happy to change them anytime.

Thanks!